addons/velero/locals.tf

```diff
@@ -34,8 +34,6 @@ locals {
     replace                    = try(var.velero_extra_configs.replace, "false")
   }
 
-  velero_extra_configs = var.velero_extra_configs
```

Comment:
`velero_extra_configs` attribute can not be removed, this is required.

Reply:
this was not used anywhere, as terraform cleaner tool.

Reply:
FYR : https://github.com/clouddrove/terraform-aws-eks-addons/blob/master/_examples/complete/variables.tf#L120-L127